### ISSUE TITLE
Salvage Fleet Modules (all of them)

### ISSUE THREAD

**Issue by ghosty1974** [goatfungus/NMSSaveEditor#334]
All of the s-class salvage fleet modules do not seem to work with any version of nms (includeing 3.05) when useing the save editor. They appear in the inventory but when you use them, they dont work at all. 
Is there a fix for this or a work around (apart from running derelict freighters).
Thx.

**Comment by ukmerlin** [comment]
It is not enough to have the element for these in storage. The game uses the source as part of its procedural name and use and appears to tie it to other uses. For example a valid module will tell which freighter it came from. I have not tried this but it might work if you instead obtain one real module, then use NMSE to increase the count of it. Let us know if this works :)

**Comment by ghosty1974** [comment]
solution so far is, get one orig module. save game, open rawJSON and litterly copy the item in inventory and paste copy the code and assign it a slot.
i tried stacking them to 10, didnt work, tbh i didnt think that would. 
side note, the module reads in the save editor as a c-class, but after several resaves and checks, it is a s-class in game.

**Comment by Novoca1n3** [comment]
Of course 10 wouldn't work. You can only have a maximum of 6 (3 in tach and 3 in inv). Any more and they all stop working.

**Comment by ukmerlin** [comment]
Novo - you are referring to installed, we are discussing how to obtain installables.

**Comment by Novoca1n3** [comment]
My apologies

**Comment by Mike-Blackwell** [comment]
Dumb question... So does one have to have a real one for all item types (7x)? does it matter what level of module e.g. "C" vs "S"? If not then Product>Procedural> Salvaged Fleet XX option creates modules with Freighter names. I noticed that when I was coping the seven types my last two appeared as green = "C" modules but in game play two of them were proper "S" not sure how or why?

**Comment by Mike-Blackwell** [comment]
Well, me thinks I found a way which is kind of clumsy but it works;
I used the Product>Procedural> "Salvaged Fleet XX" Option it creates green ="C" whilst in the Save Editor make a number of each. Save and then start the game and harvest the modules the are "S" Class (move them to your exosuit) it seems like I get one every 8-10th one. Then go back into Save Editor get rid of the non "S" modules and repopulate and continue to go back to the game and "rinse & Repeat" until you get all 7 "S" Modules. Time consuming but works then I simply do dup 7x (one for storage and the other 6 for insertion).
